crates/eips/src/eip2930.rs

```diff
@@ -101,6 +109,41 @@ impl AccessList {
         })
     }
 
+    /// Returns the position of the given address in the access list, if present.
+    pub fn index_of_address(&self, address: Address) -> Option<usize> {
+        self.iter().position(|item| item.address == address)
+    }
+
+    /// Checks if a specific storage slot within an account is present in the access list.
+    ///
+    /// Returns a tuple with flags for the presence of the account and the slot.
+    pub fn contains(&self, address: Address, slot: B256) -> (bool, bool) {
```

Comment:
imo mirror other method so `contains_storage`? if not, for me the immediate expectation was that `contains` is what `contains_address` is